php/classes/controllers/class-extensions-controller.php

```diff
@@ -113,13 +114,25 @@ public function render_seriously_simple_extensions() {
 				'new_window'  => true,
 				'button_text' => __( 'Get Second Line Themes', 'seriously-simple-podcasting' ),
 			),
+			'elementor-templates' => array(
```

Comment:
As discussed, let's move the Elementor Templates import extension to just after the Castos extension, and only show if if Elementor is installed.